std_msgs/Header header 
int32 sonar_front
int32 sonar_left
int32 sonar_right